components/automate-ui/src/app/components/page-picker/page-picker.component.html

```diff
@@ -1,4 +1,4 @@
-<ng-container *ngIf="selectablePages.length > 1">
+<ng-container *ngIf="selectablePages.length > 1 || last > 1">
```

Comment:
It's possible that sometimes we have a scenario with only 1 selectable page, however we're looking at say...page 11.  This makes sure that if we have only one selectable page, but more than 1 page in total, we still see the pagination component.

@msorens - thinking now that I probably need to pipe this, huh?

Reply:
No, using properties in expressions local to the template like this is fine; it is only when you introduce a function call that a pipe would improve performance.